tests/Microsoft.Identity.Web.AotCompatibility.TestApp/Microsoft.Identity.Web.AotCompatibility.TestApp.csproj

```
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <PublishAot>true</PublishAot>
```

```suggestion
    <PublishAot>true</PublishAot>
    <EventSourceSupport>true</EventSourceSupport>
```

We should also set this property. This is something I learned recently. EventSourceSupport is false by default in PublishAot, and so it won't produce warnings hidden behind IsEnabled() checks in EventSource.

---

Done